addons/l10n_in_hr_holidays/views/hr_leave_views.xml

```diff
@@ -10,7 +10,10 @@
                 <field name="l10n_in_contains_sandwich_leaves" invisible="1" />
             </xpath>
             <xpath expr="//field[@name='duration_display']" position="after">
-                <div name="l10n_in_contains_sandwich_leaves" invisible="not l10n_in_contains_sandwich_leaves" class="w-md-75 float-end ps-3 alert alert-warning" role="alert">
+                <div name="l10n_in_contains_sandwich_leaves"
+                     invisible="leave_type_request_unit == 'hour' or request_unit_half or not l10n_in_contains_sandwich_leaves"
```

Comment:
Hi, thanks for your comment
when we checked request_unit_half its considered as a half day otherwise its a full day.
Therefore, I added them separately.

Reply:
You're right :+1: